frontend/src/static/js/v_authorship.js

```diff
@@ -184,6 +184,26 @@ window.vAuthorship = {
       }
     },
 
+    getFileBlameLink(file) {
+      const repo = window.REPOS[this.info.repo];
+
+      return `http://github.com/${
+        repo.location.organization}/${
+        repo.location.repoName}/blame/${
+        repo.branch}/${
+        file.path}`;
+    },
+
+    getFileHistoryLink(file) {
+      const repo = window.REPOS[this.info.repo];
+
+      return `http://github.com/${
+        repo.location.organization}/${
+        repo.location.repoName}/commits/${
+        repo.branch}/${
+        file.path}`;
+    },
```

Comment:
i would do a combined one
```
getFileLink(file, path) {
      const repo = window.REPOS[this.info.repo];

      return `http://github.com/${
        repo.location.organization}/${
        repo.location.repoName}/${path}/${
        repo.branch}/${
        file.path}`;
    },
```

then you can call with `getFileLink(file, 'blame')` and `getFileLink(file, 'commits')`